src/controllers/CloudFilesController.cpp

```diff
@@ -105,7 +105,9 @@ CloudFilesModel *Self::model()
 
 void Self::switchToRootFolder()
 {
-    switchToHierarchy({ rootFolder() });
+    if (FileUtils::forceCreateDir(rootFolder()->localPath())) {
```

Comment:
This line will will case `abort` if folder will not be created

Reply:
This code is outdated now